cli/src/command/generate_config.rs

```diff
@@ -104,30 +102,50 @@ impl GenerateConfig {
     }
 
     /// If no port is provided by user, uses `DEFAULT_EXONUM_LISTEN_PORT`.
-    fn parse_external_address(input: &str) -> Result<SocketAddr, Error> {
-        if let Ok(address) = input.parse() {
-            Ok(address)
-        } else {
-            let ip_address = input.parse()?;
-            Ok(SocketAddr::new(ip_address, DEFAULT_EXONUM_LISTEN_PORT))
+    fn resolve_peer_address(
+        hostname: &str,
+        is_host_with_port: &mut bool,
```

Comment:
Nit: Why not pass this var by value, rather than by a mutable reference? A mutable reference could make sense if the var is read after the call, but this is seemingly not the case.

Reply:
Influence of C 😄